### PR TITLE
Add method to get widget attributes

### PR DESCRIPTION
**Pull request by pxpm**
`Widget` attributes are a protected property. 

We needed an acessor. I build the accessor. 

debatable `get` vs `getAttribute`. For me `get` seems more like to `get a widget` than `get an attribute of the widget` .. so `getAttribute` for me 🙏

### CODE REVIEW

File: src/app/Library/Widget.php [hunk 1]
```diff
@@ -48,6 +48,17 @@ public static function add($attributes = null)
...
+     * Return the widget attribute value or null when attribute does not exist.
+     *
+     * @param  string  $attribute
+     * @return mixed
+     */
+    public function getAttribute(string $attribute)
+    {
+        return $this->attributes[$attribute] ?? null;
+    }
+
```
Comment (tabacitu) [inline src/app/Library/Widget.php:61]: I noticed `getAttribute()` might returning NULL in two cases:
- (1) when the attribute is `null`
- (2) when the attribute doesn't exist

It's the same problem we had with the `request()`, remember? No way to differentiate if an input is _actually_ null or just doesn't exit.

I think we'll have that confusion pretty soon, so what do you think about also adding:

```suggestion

    /**
     * Check if the widget has an attribute.
     *
     * @param  string  $attribute
     * @return bool
     */
    public function hasAttribute(string $attribute)
    {
        return array_key_exists($attribute, $this->attributes);
    }
```

PS. Did NOT test the code above - you please do if you agree. But it should work, right?

**Review by tabacitu — CHANGES_REQUESTED** [review]
I love it! ❤️

So small and clean, no reason not to merge it, even if it didn't help in other PRs.

Ok fine, one reason, but it's small 😅 and might be a premature optimization. Check it out please 👇

File: src/app/Library/Widget.php (hunk 1 above)
Comment (pxpm) [reply src/app/Library/Widget.php:61]: Totally, just updated PR!

**Comment by tabacitu** [comment]
This might be our cleanest PR yet 😅 I wish all of them were like this ❤️

**Pull request merged**